includes/bootstrap.inc

```diff
@@ -273,7 +274,7 @@ function drush_bootstrap_validate($phase) {
         break;
       }
       if ($phase_index > $validated_phase) {
-        $current_phase .= '_validate';
+        $current_phase .= 'Validate';
```

Comment:
I'm a bit concerned about the BC implications of changing bootstrap hook names. Maybe c'est la vie, but I just want to discuss.